### PR TITLE
feat(pager): add align to sides

### PR DESCRIPTION
**Pull request by sendilkumarn**
#208

### CODE REVIEW

File: src/pager/pager.spec.ts [hunk 1]
```diff
@@ -131,6 +145,23 @@ describe('ngb-pagination', () => {
...
+         tcb.overrideTemplate(TestComponent, html).createAsync(TestComponent).then((fixture) => {
+           fixture.detectChanges();
+           expectPager(fixture.nativeElement, [true, true]);
+           expectAlignClasses(fixture.nativeElement, [true, true]);
+
+           fixture.componentInstance.noOfPages = 5;
+           fixture.componentInstance.page = 5;
+           fixture.detectChanges();
+           expectPager(fixture.nativeElement, [true, false]);
+           expectAlignClasses(fixture.nativeElement, [true, true]);
```
Comment (pkozlowski-opensource) [inline src/pager/pager.spec.ts]: I don't think there is much point in having a separate `expectAlignClasses` method. Just check the presence of CSS classes directly here in this test. Or just create a simpler helper method.

File: src/pager/pager.spec.ts [hunk 2]
```diff
@@ -131,6 +145,23 @@ describe('ngb-pagination', () => {
            expectPager(fixture.nativeElement, [true, true]);
          });
        })));
+
+    it('should have pager-prev and pager-next classes set', async(inject([TestComponentBuilder], (tcb) => {
+         const html = '<ngb-pager [noOfPages]="noOfPages" page="5" [alignLinks]=true></ngb-pager>';
```
Comment (pkozlowski-opensource) [inline src/pager/pager.spec.ts]: `[alignLinks]=true>` => `[alignLinks]="true">`

File: src/pager/pager.spec.ts (hunk 1 above)
Comment (sendilkumarn) [inline src/pager/pager.spec.ts]: noted. Thought of using it elsewhere. Okay will change it 👍

File: src/pager/pager.spec.ts [hunk 3]
```diff
@@ -131,6 +145,23 @@ describe('ngb-pagination', () => {
...
+
+         tcb.overrideTemplate(TestComponent, html).createAsync(TestComponent).then((fixture) => {
+           fixture.detectChanges();
+           expectPager(fixture.nativeElement, [true, true]);
```
Comment (pkozlowski-opensource) [inline src/pager/pager.spec.ts]: We don't need to test for this, already covered in other tests.

File: src/pager/pager.spec.ts [hunk 4]
```diff
@@ -131,6 +145,23 @@ describe('ngb-pagination', () => {
...
+           expectAlignClasses(fixture.nativeElement, [true, true]);
+
+           fixture.componentInstance.noOfPages = 5;
+           fixture.componentInstance.page = 5;
+           fixture.detectChanges();
+           expectPager(fixture.nativeElement, [true, false]);
```
Comment (pkozlowski-opensource) [inline src/pager/pager.spec.ts]: This section of the above 4 lines is not needed, as already covered in other tests. What we should have instead is a change in the `alignLinks` binding to see that CSS classes are added / removed.

Comment (sendilkumarn) [inline src/pager/pager.spec.ts]: toggle align?  Think adding a function to handle alignLink is not needed right?

Comment (pkozlowski-opensource) [inline src/pager/pager.spec.ts]: What I meant is that: `<ngb-pager [noOfPages]="noOfPages" page="5" [alignLinks]="shouldAlign">` and then in this test we should flip it between true / false and verify that the corresponding CSS classes are added / removed.

**Comment by pkozlowski-opensource** [comment]
Merged, thnx! In the future please squash commits into one and add `Fixes #xyz` to indicate which issue is being fixed by the commit.

**Pull request closed without merging**